src/User.ts

```diff
@@ -72,19 +72,28 @@ export class User {
     })
   }
 
-  public signMsg (rawMsg: string): Promise<string> {
-    return new Promise<string>((resolve, reject) => {
+  public signMsg (msgHash: string): Promise<any> {
```

Comment:
Should be called signMsgHash if it takes the Hash

Reply:
yes makes sense, changing it